### DISCUSSION TITLE
Can Prisma be used in a browser environment?

### DISCUSSION THREAD

**Discussion by vjpr** [prisma/prisma#5989] [Q&A] [answered]
I need an SQL database that will run in both a Chrome Extension and an Electron app.

I am planning to use: [SQLite.js](https://github.com/sql-js/sql.js).

Since Prisma has a native Rust component, I'm guessing this is not possible.

How easy would it be to deploy Prisma cross-platform along with an Electron app?

If I wanted to add SQLite.js support to Prisma, would this even be possible - is there a layer before Rust that an adapter can be built?

If I want to add any new database adapter, I must code the interface in Rust - is this correct?

**Comment by ryands17** [comment, marked as answer]
@vjpr 👋 
Yes you're correct. Currently Prisma cannot be used in a browser environment and only in Node. in Electron, you can use it in the Node API and then send the results via IPC.

As for SQLite.js support, @pimeys might be the best one to answer this :)

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
